tests/sync/app.cpp

```diff
@@ -1300,9 +1327,21 @@ TEST_CASE("app: remote mongo client", "[sync][app]") {
             CHECK(!error);
             auto breed = static_cast<std::string>((*document)["breed"]);
             CHECK(breed == "king charles");
+        });
+        
+        dog_collection.find_one_and_update({{"name", "invalid name"}}, {{"name", "some name"}}, [&](Optional<bson::BsonDocument> document, Optional<app::AppError> error) {
+            CHECK(!error);
+            CHECK(!document);
             processed = true;
         });
         
+//        dog_collection.find_one_and_update({{"name", "invalid name"}}, {{}}, find_and_modify_options, [&](Optional<bson::BsonDocument> document, Optional<app::AppError> error) {
```

Comment:
Remove if not needed

Reply:
I believe this test is being commented out like the above due to a bug server side.

Reply:
Maybe a `FIXME: enable once server bug is fixed` is fine then :-)